plugins/woocommerce/tests/e2e-pw/tests/merchant/order-edit.spec.js

```diff
@@ -42,16 +42,20 @@ test.describe( 'Edit order', () => {
 	} );
 
 	test( 'can view single order', async ( { page } ) => {
-		// go to orders page
-		await page.goto( 'wp-admin/edit.php?post_type=shop_order' );
+		if ( process.env.DISABLE_HPOS === 1 ) {
+			await page.goto( '/wp-admin/admin.php?page=wc-orders&action=new' );
+		} else {
+			await page.goto( 'wp-admin/edit.php?post_type=shop_order' );
+		}
```

Comment:
Testing with `DISABLE_HPOS`, this seems to be the only place where things don't redirect properly/are sensitive to HPOS.

Reply:
Shouldn't this one be the other way around?

Reply:
@lanej0 in case you missed the ☝🏼 notification.

Reply:
Oh geez, yeah.  I totally reversed that.